### PR TITLE
add workflowFunction facades config

### PR DESCRIPTION
**Pull request by Thiblef**
https://github.com/open-orchestra/open-orchestra-base-api-bundle/pull/67
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1288
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/101

### CODE REVIEW

File: WorkflowFunctionAdminBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+namespace OpenOrchestra\WorkflowFunctionAdminBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files
+ *
+ * To learn more see {
```
Comment (nicolasThal) [inline WorkflowFunctionAdminBundle/DependencyInjection/Configuration.php]: doc to update

**Comment by nicolasThal** [comment]
:+1: with the doc

File: WorkflowFunctionAdminBundle/Transformer/WorkflowFunctionCollectionTransformer.php [hunk 2]
```diff
@@ -20,7 +19,7 @@ class WorkflowFunctionCollectionTransformer extends AbstractSecurityCheckerAware
      */
     public function transform($mixed)
     {
-        $facade = new WorkflowFunctionCollectionFacade();
+        $facade = $this->newFacade();
```
Comment (itkg-ngilain) [inline WorkflowFunctionAdminBundle/Transformer/WorkflowFunctionCollectionTransformer.php]: @Thiblef : Can you throw an exception ``if (!$facade instanceof FacadeInterface)``

File: WorkflowFunctionAdminBundle/Transformer/WorkflowFunctionTransformer.php [hunk 3]
```diff
@@ -33,7 +37,7 @@ public function __construct(TranslationChoiceManager $translationChoiceManager,
      */
     public function transform($mixed)
     {
-        $facade = new WorkflowFunctionFacade();
+        $facade = $this->newFacade();
```
Comment (itkg-ngilain) [inline WorkflowFunctionAdminBundle/Transformer/WorkflowFunctionTransformer.php]: @Thiblef : Can you throw an exception ``if (!$facade instanceof FacadeInterface)``

**Comment by ngilain** [comment]
:+1:

File: WorkflowFunctionAdminBundle/DependencyInjection/Configuration.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('open_orchestra_workflow_function_admin');
+
+        $rootNode->children()
+            ->arrayNode('transformer')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->arrayNode('workflow_function')
```
Comment (alavieille) [inline WorkflowFunctionAdminBundle/DependencyInjection/Configuration.php]: same remark than in cms bundle

**Comment by Thiblef** [comment]
@itkg-alavieille @nicolasThal @itkg-ngilain

**Comment by alavieille** [comment]
:+1:

**Comment by ngilain** [comment]
:+1:

**Comment by alavieille** [comment]
:+1:

**Pull request merged**